### PR TITLE
[Collections] Add delegate method to check whether swiping should be limited to one direction.

### PR DESCRIPTION
**Pull request by randallli**
[Collections] Add delegate method to check whether swiping should be limited to one direction.

### CODE REVIEW

File: components/NavigationDrawer/examples/BottomDrawerExpandFullscreenExample.swift [hunk 1]
```diff
@@ -17,7 +17,7 @@ import MaterialComponents.MaterialBottomAppBar
 import MaterialComponents.MaterialBottomAppBar_ColorThemer
 import MaterialComponents.MaterialColorScheme
 import MaterialComponents.MaterialNavigationDrawer
-import MaterialComponents.MaterialNavigationDrawer_ColorThemer
```
Comment (rami-a) [inline components/NavigationDrawer/examples/BottomDrawerExpandFullscreenExample.swift]: These changes appear unrelated, perhaps they should be split out into a separate PR

File: contributing/bazel_kokoro.md [hunk 2]
```diff
@@ -117,22 +117,19 @@ If your component depends on non-source resources, you can add a bundle target t
 with the following additions:
 
 ```
+load("//tools/build_defs/apple:resources.bzl", "apple_bundle_import")
+
 mdc_public_objc_library(
     name = "ComponentName",
-    bundles = [":Bundle"],
```
Comment (rami-a) [inline contributing/bazel_kokoro.md:122]: These changes appear unrelated, perhaps they should be split out into a separate PR

File: .gitattributes [hunk 3]
```diff
@@ -1 +1,2 @@
-snapshot_test_goldens/**/*.png filter=lfs diff=lfs merge=lfs -text
+# Do not add git-lfs attributes to this files. It causes problems with Cocoapods.
```
Comment (rami-a) [inline .gitattributes:1]: We do not want this change, looks like a carry-over from stable branch

**Comment by randallli** [comment]
moved this into https://github.com/material-components/material-components-ios/pull/6170

File: components/NavigationDrawer/examples/BottomDrawerExpandFullscreenExample.swift (hunk 1 above)
Comment (randallli) [reply components/NavigationDrawer/examples/BottomDrawerExpandFullscreenExample.swift]: Removed. this was an artifact of the export that I have now fixed.

File: contributing/bazel_kokoro.md (hunk 2 above)
Comment (randallli) [reply contributing/bazel_kokoro.md:122]: https://github.com/material-components/material-components-ios/pull/6171

File: .gitattributes (hunk 3 above)
Comment (randallli) [reply .gitattributes:1]: Done. in other pr by rebasing.
